fastlane/lib/fastlane/actions/create_keychain.rb

```diff
@@ -22,31 +24,31 @@ def self.run(params)
         end
 
         commands = []
-        commands << Fastlane::Actions.sh("security create-keychain -p #{escaped_password} #{keychain_path}", log: false)
+        commands << Fastlane::Actions.sh("security create-keychain -p #{escaped_password} #{keychain_path}", log: should_log)
 
         if params[:default_keychain]
           # if there is no default keychain - setting the original will fail - silent this error
           begin
-            Actions.lane_context[Actions::SharedValues::ORIGINAL_DEFAULT_KEYCHAIN] = Fastlane::Actions.sh("security default-keychain", log: false).strip
+            Actions.lane_context[Actions::SharedValues::ORIGINAL_DEFAULT_KEYCHAIN] = Fastlane::Actions.sh("security default-keychain", log: should_log).strip
           rescue
           end
-          commands << Fastlane::Actions.sh("security default-keychain -s #{keychain_path}", log: false)
+          commands << Fastlane::Actions.sh("security default-keychain -s #{keychain_path}", log: should_log)
         end
 
-        commands << Fastlane::Actions.sh("security unlock-keychain -p #{escaped_password} #{keychain_path}", log: false) if params[:unlock]
+        commands << Fastlane::Actions.sh("security unlock-keychain -p #{escaped_password} #{keychain_path}", log: should_log) if params[:unlock]
```

Comment:
Shell commands that contain passwords should ideally never have the possibility of making it into logs since often users can't control security for those logs (on hosted CI, for example).